### DISCUSSION TITLE
Sorting question (file extension + set order)

### DISCUSSION THREAD

**Discussion by Enferlain** [d2phap/ImageGlass#1521] [General]
Just wanted to ask if there was a way to set imageglass to go through files as they are next to each other if I have them grouped by type and sorted by date in the explorer? So if I press left or right to move to next/prev pic, it won't jump between the extensions but go through them in order as they appear in file explorer?

**Comment by d2phap** [comment]
ImageGlass has limited support for File Explore's sort order (by name, file size, extension). And it does not support for grouping.
